webapp/components/lobby_creator/creator.tsx

```diff
@@ -0,0 +1,35 @@
+import Socket from "@/client/socket"
+import React, { FormEvent } from "react"
+import styles from './creator.module.scss'
+
+interface CreatorProps {
+    socket:Socket
+}
+
+const Creator : React.FC<CreatorProps> = ({socket}) => {
```

Comment:
Maybe call it `LobbyCreator` or `LobbyCreationForm`? Just `Creater` is a little confusing at first glance

Reply:
ok